critiquebrainz/frontend/external/musicbrainz_db/tests/release_group_test.py

```diff
@@ -12,9 +12,10 @@ def setUp(self):
         setup_cache()
         mb_release_group.mb_session = MagicMock()
         self.mock_db = mb_release_group.mb_session.return_value.__enter__.return_value
-        self.release_group_query = self.mock_db.query.return_value.options.return_value.\
-                options.return_value.options.return_value.options.return_value.filter.return_value.all
-        self.release_group_query_without_artists = self.mock_db.query.return_value.options.return_value.filter.return_value.all
+        self.release_group_query = self.mock_db.query.return_value.options.return_value.options.return_value.\
+            options.return_value.options.return_value.options.return_value.filter.return_value.all
+        self.release_group_query_without_artists = self.mock_db.query.return_value.\
+            options.return_value.options.return_value.filter.return_value.all
```

Comment:
I'm not sure if it's any better. It would be useful if you could describe how you wrote this and how/when it should be modified in the future.